src/logger/morgan/morgon.logger.js

```diff
@@ -0,0 +1,16 @@
+import morgan from "morgan";
+import logger from "../winston/index.js";
+
+const morganMiddleware = morgan(
+  ":method :url :status :res[content-length] - :response-time ms",
+  {
+    stream: {
+      write: (message) => {
+        logger.http(message.trim());
+      },
+    },
+    skip: (req, res) => res.statusCode < 400,
```

Comment:
Log all the api logs do not skip success responses. I think you can remove this `skip` key.